products/caasp/main.pm

```diff
@@ -134,8 +134,9 @@ sub load_stack_tests {
         loadtest 'caasp/stack_configure';
         loadtest 'caasp/stack_bootstrap';
         loadtest 'caasp/stack_kubernetes';
-        loadtest 'caasp/stack_update'      if update_scheduled;
-        loadtest 'caasp/stack_add_nodes'   if get_delayed_worker;
+        loadtest 'caasp/stack_update'    if update_scheduled;
+        loadtest 'caasp/stack_add_nodes' if get_delayed_worker;
+        loadtest 'caasp/helm';
```

Comment:
Please rename to `caasp/stack_helm` - just to separate CaaSP and MicroOS testsuites